department-of-veterans-affairs/caseflow
# Realistic name seeds

### Description
Seed data updated to give test data users with permissions or test data realistic names. 

Before: 
`Role Description`
e.g. `Judge with hearings and cases`

After:
`GivenName RoleDescription FamilyName`
e.g. `Aaron JudgeWithHearingsAndCases Abshire`

### Acceptance Criteria
- [ ] Code still works
- [ ] Other team members like the change

### Testing Plan
1. Go to [`/test/users`](http://localhost:3000/test/users), look over the names. 
1. Browse as various users and see if you prefer the UI
   - Attorney list on judge Assign
   - Judge list on reassign to judge
   - Names on case details page
   - Names on hearings schedule & details
   - Other locations too, probably? Everywhere we show users names!

### User Facing Changes
Dev only
